pages/en/billing.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: Billing on Subgraph Studio
+---
+
+### Overview
+
+For a quick demo of how billing works on the Subgraph Studio, check out the video below:
+
+<VideoEmbed youtube="UrfIpm-Vlgs" />
+
+Invoices are statements of payment amounts owed by a customer and are typically generated on a weekly basis in the system. You'll be required to pay fees based on the query fees you generate using your API keys. While The Graph protocol operates on Ethereum Mainnet, the billing contract lives on the [Polygon](https://polygon.technology/) network to reduce transaction times and cost. It'll allow you to:
+
+- Add and remove GRT from your wallet
+- Keep track of your balances based on how much GRT you have added to your account, how much you have removed, and your invoices
+- Automatically clear payments based on query fees generated
```

Comment:
Not sure what this means: "clear payments". Maybe: automatically pay invoices based on query fees generated, as long as there is enough billing balance in your account.

Reply:
Yes, that's 100% better - good call!